### PR TITLE
Update multitech.sc

### PR DESCRIPTION
**Pull request by Pietr06**
added /player and gamerule sleeping player percentage

### CODE REVIEW

File: multitech.sc [hunk 1]
```diff
@@ -50,6 +50,9 @@ uninstall() -> (
 );
 
 install(creative) -> for([
+    
+    _(c)->'/gamerule playerSleepengPercentage 0',
```
Comment (BisUmTo) [inline multitech.sc]: Wrong spelled

**Review by BisUmTo — CHANGES_REQUESTED** [review]

File: multitech.sc [hunk 2]
```diff
@@ -66,6 +69,7 @@ install(creative) -> for([
     _(c)->'/carpet setDefault updateSuppressionCrashFix true',
     _(c)->'/carpet setDefault commandScript ops',
     _(c)->if(c,'/carpet setDefault liquidDamageDisabled true'),
+    _(c)->'/carpet setdefault commandPlayer true',
```
Comment (BisUmTo) [inline multitech.sc]: set default with D

**Review by BisUmTo — CHANGES_REQUESTED** [review]

File: multitech.sc [hunk 3]
```diff
@@ -50,6 +50,9 @@ uninstall() -> (
 );
 
 install(creative) -> for([
+    
+    _(c)->'/gamerule playerSleepingPercentage 0',
```
Comment (BisUmTo) [inline multitech.sc]: playersSleepingPercentage

**Review by BisUmTo — CHANGES_REQUESTED** [review]

File: multitech.sc (hunk 3 above)
Comment (Pietr06) [reply multitech.sc]: dove sta il problema mo xd

Comment (BisUmTo) [reply multitech.sc]: players e non player

Comment (Pietr06) [reply multitech.sc]: ahh

**Comment by Pietr06** [comment]
ahahhaha ce l'ho fatta

**Pull request merged**